logictest/runner.go

```diff
@@ -105,10 +105,10 @@ func (r *Runner) handleCreateTable(ctx context.Context, c *datadriven.TestData)
 	var schema *dynparquet.Schema
 	for _, arg := range c.CmdArgs {
 		if arg.Key == "schema" {
-			if len(arg.Vals) != 1 && arg.Vals[0] != "default" {
+			if len(arg.Vals) != 1 {
 				return "", fmt.Errorf("createtable: unexpected schema values %v", arg.Vals)
 			}
-			schema = r.defaultSchema
+			schema = r.schemas[arg.Vals[0]]
```

Comment:
nit: return an error if schema is not found

Reply:
We do below where we check if schema is `nil`